ACMAS/app/ACMAS_Web/templates/index.html

```diff
@@ -14,20 +14,97 @@
 {% block facebook_description %}ACMAS is the free homework and backtest helper created by students for students. Join our community of students anonymously today!{% endblock %}
 {% block facebook_url %}www.acmas.systems{% endblock %}
 
-
 {% block content %}
-<div class="flex flex-wrap mt-24 mx-auto max-w-screen-xl">
-    <div class="text-7xl" style="color: #327796; font-family: Inter, serif;">Simplify <br> studying with</div>
-    <div class="">
-        <a href="/">
-            <img
-                alt="ACMAS logo"
-                class="w-96 h-auto pt-4 pl-5"
-                src="../static/img/ACMAS_Logo.svg"
-            />
-        </a>
+<div class="p-0 flex items-center overflow-hidden relative" style="height: 75vh;">
+    <div class="absolute w-full h-full" style="background: linear-gradient(-45deg, #f8f8f3, #f5f2d6, #dbebf3, #c6d6f7); background-size: 300% 300%; animation: gradient 65s ease infinite;"> </div>
+    <div class="container mx-auto px-4 z-3 pt-6 relative lg:w-6/12">
+        <div class="justify-center flex flex-wrap flex-col -mx-4 text-center">
+            <h1 class="text-7xl font-bold text-white leading-tight mt-0" style="color: #327796">ACMAS</h1>
```

Comment:
Replace with ACMAS folder logo

Reply:
Does this work? 
![image](https://github.com/ACMAS/ACMAS-Frontend/assets/31413744/6ae92e05-5e4f-43cf-ad03-09ab41045b84)

Reply:
Perfect!